pymc/distributions/multivariate.py

```diff
@@ -929,7 +929,7 @@ class Wishart(Continuous):
 
     @classmethod
     def dist(cls, nu, V, *args, **kwargs):
-        nu = pt.as_tensor_variable(intX(nu))
+        nu = pt.as_tensor_variable(nu, dtype=int)
```

Comment:
What will this default to on the `pytensor` side?

Reply:
The same that numpy does, i.e., int64.